src/datachannel.cpp

```diff
@@ -170,7 +172,7 @@ void DataChannel::open(shared_ptr<SctpTransport> transport) {
 }
 
 void DataChannel::processOpenMessage(message_ptr) {
-	PLOG_WARNING << "Received an open message for a user-negotiated DataChannel, ignoring";
```

Comment:
I think it would be good to keep the log line as debug.

Reply:
Instead of a counter?

Reply:
No, in addition to it. This message can only be sent over DTLS by a trusted peer anyway.